.github/ISSUE_TEMPLATE/meeting-team.md

```diff
@@ -12,7 +12,7 @@ This is a planning issue for our next team meeting! See below for information ab
 _Information about when and where the meeting will take place_
 
 - **Date:** <YYYY-MM-DD>
-- **Time:** 7:30AM US/Pacific time (or [**your timezone**](https://arewemeetingyet.com/Los%20Angeles/<YYYY-MM-DD>/07:30/2i2c%20Team%20Meeting#eyJ1cmwiOiJodHRwczovL2hhY2ttZC5pby9ZNVNCTXhWN1I2Q01xemVUWGdtNWtBIn0=))
+- **Time:** 8:30AM US/Pacific time (or [**your timezone**](https://arewemeetingyet.com/Los%20Angeles/<YYYY-MM-DD>/08:30/2i2c%20Team%20Meeting#eyJ1cmwiOiJodHRwczovL2hhY2ttZC5pby9ZNVNCTXhWN1I2Q01xemVUWGdtNWtBIn0=))
```

Comment:
These `arewemeetingyet` links don't seem to work anymore without a date 😕 
![undefined](https://user-images.githubusercontent.com/7579677/160375758-2f8c1d0f-a839-4a5b-9dc2-efc154b9d523.png)

Reply:
Ping @choldgraf on that - I feel like he got it to work on one point?

Reply:
argh no I could never figure that out. Didn't we recently discuss **just** using the Google Calendar for this, and having a reminder about it show up in our slacks, rather than using a GitHub issue? I would be fine just using the calendar entry as the source of truth rather than also including the time in our issue template. WDYT?

Reply:
Yep, this is totally confusing... having everything on the calendar should help with the SSoT idea...